routes/index.tsx

```diff
@@ -1,38 +1,74 @@
-import { Handlers, PageProps } from "$fresh/server.ts"
-import { Footer } from "@/components/Footer.tsx"
-import { HeadElement } from "@/components/HeadElement.tsx"
-import { Header } from "@/components/Header.tsx"
-import IconCart from "@/components/IconCart.tsx"
-import { formatBalance, sanitizeUri } from "@/utils/data.ts"
-import { graphql } from "@/utils/indexer.ts"
-import { Item } from "@/utils/types.ts"
-import { useComputed } from "@preact/signals"
-import { aspectRatio } from "@twind/aspect-ratio"
-import { extendFields, getClient } from '@kodadot1/uniquery'
-import { tw } from "twind"
-
-
-const client = getClient()
-const collectionId = Deno.env.get('COLLECTION_ID') || '30'
-const { query: q } = client.itemListByCollectionId(collectionId, {
-  fields: extendFields(['meta', 'price']),
-  orderBy: 'createdAt_ASC',
-})
+import { Handlers, PageProps } from "$fresh/server.ts";
+import { Footer } from "@/components/Footer.tsx";
+import { HeadElement } from "@/components/HeadElement.tsx";
+import { Header } from "@/components/Header.tsx";
+import IconCart from "@/components/IconCart.tsx";
+import { formatBalance, sanitizeUri } from "@/utils/data.ts";
+import { graphql } from "@/utils/indexer.ts";
+import { Item } from "@/utils/types.ts";
+import { useComputed } from "@preact/signals";
+import { aspectRatio } from "@twind/aspect-ratio";
+import { extendFields, getClient } from "@kodadot1/uniquery";
+import { tw } from "twind";
+
+const client = getClient("ahp");
+const collectionId = Deno.env.get("COLLECTION_ID") || "11";
+
+interface ItemsResult {
+  items: Item[];
+}
+
+interface CountResult {
+  itemCount: {
+    totalCount: number;
+  };
+}
 
 interface Data {
   items: Item[];
+  page: number;
+  pageCount: number;
 }
 
 export const handler: Handlers<Data> = {
-  async GET(_req, ctx) {
-    const data = await graphql<Data>(q);
+  async GET(req, ctx) {
+    const url = new URL(req.url);
+
+    const countQuery = client.itemCountByCollectionId(collectionId);
+    const count = await graphql<CountResult>(
+      countQuery.query,
+      countQuery.variables,
+    )
+      .then((result) => result?.itemCount?.totalCount);
+
+    const itemsPerPage = 12;
```

Comment:
Also would put this to ENV as `PAGINATION`

Reply:
done